### ISSUE TITLE
CPU forbruget på itf2-GIS

### ISSUE THREAD

**Issue by adam42-dk** [ballerupgis/SeMaWi#241]
Hej

Jeg har nu forsøgt, at sætte en dev-version op af SeMaWi på itf2-GIS serveren, men er stødt ind i et "mindre" problem / udfordring:

Når jeg kører kommandoen docker stats kan jeg se at containeren postgis (id 85d04867af64) optager ret meget af CPU kræfterne - den står og svinger mellem 80 og 104 % i forbruget.

Det gør, at det er ret svært at sætte noget nyt op og kører det andre containerne med en rimelig styrke.

Enten skal vi have sat et loft på forbruget af den process eller også finde ud af hvordan vi får den container optimeret. Hvad siger du @baffioso ?

**Comment by baffioso** [comment]
@mapcentia er det noget du vil kigge på vedr. Postgis container og forbrug?

**Comment by fadeto404** [comment]
Det ser ikke ud som om at der er et problem lige pt.
Hverken ```docker stats``` eller ```htop``` viser et CPU-forbrug der er problematisk (0.5-15%).

Vi kan evt. prøve at stressteste databasen, og se om det giver problemer, men ellers tænker jeg bare at vi fortsætter med at sætte op.

**Comment by mapcentia** [comment]
Det kan være backup/sync, som strækker sig til op af dagen. Jeg vil lige tjekke, hvad det er PostgreSQL laver om morgenen.

**Comment by adam42-dk** [comment]
@fadeto404 ja, problemet er der ikke lige nu, så det kan være at der er sat en backup/sync på et "forkert" tidspunkt, som @mapcentia antyder :) 

#afventer
